web/pgadmin/tools/sqleditor/static/js/components/sections/ResultSetToolbar.jsx

```diff
@@ -142,13 +143,30 @@ export function ResultSetToolbar({query,canEdit, totalRowCount}) {
     });
   }, []);
 
+  const byteaDataTypeCellSelected = (rows, cols, range, cell, columns) =>{
```

Comment:
Why do you want to make things so complex? This can make things slower.
You can simply change the bytea editor which will never open.

Reply:
The bytea editor is not editable so it does not open, but this function will disable the copy button, as bytea data type must be know before disabling copy button, so passing the values.

Reply:
We already had discussed that we can allow copying. There is no harm in allowing. We don't want to affect the performance just to disallow copy.

Reply:
Done